### PR TITLE
Put a thread dump into the log window for exceptions

### PR DESCRIPTION
**Pull request by outofculture**
helps us track down #246

### CODE REVIEW

File: acq4/util/LogWindow.py [hunk 1]
```diff
@@ -677,6 +670,18 @@ def formatTracebackForHTML(self, tb):
             cleanLines.append(line)
         return ''.join(cleanLines)
 
+    def formatThreadsForHTML(self, entry):
+        threads = entry["threads"]
+        traceback.format_stack
```
Comment (campagnola) [inline acq4/util/LogWindow.py:675]: ?

File: acq4/util/debug.py [hunk 2]
```diff
@@ -4,10 +4,11 @@
 Copyright 2010  Luke Campagnola
 Distributed under MIT/X11 license. See license.txt for more information.
 """
-from __future__ import print_function
 
-from pyqtgraph.debug import *
+import sys
+
 import pyqtgraph.debug as pgdebug
+from pyqtgraph.debug import *
```
Comment (campagnola) [inline acq4/util/debug.py:11]: Better to pick a specific list of names to import?

Comment (outofculture) [reply acq4/util/debug.py:11]: I agree! The actual use this is getting though is that a bunch of other places are `from acq4.debug import THING_IMPORTED_FROM_PG`, and I didn't want to track down all of those.

Comment (outofculture) [reply acq4/util/debug.py:11]: (I was just moving imports around in this file)

Comment (campagnola) [reply acq4/util/debug.py:11]: Ahaaaa, yeah it does look more like something I would do.

**Pull request merged**